<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LPT-1189" failures="1" tests="1" name="TestonEdge" time="69.472" errors="0" timestamp="2023-05-09T19:03:29 IST">
  <testcase classname="TestCases.E2E_negativeTest" name="placeOrder" time="54.819">
    <failure type="java.lang.AssertionError" message="Assertion on Principal amount is not successfull expected [10977] but found [8713]">
      <![CDATA[java.lang.AssertionError: Assertion on Principal amount is not successfull expected [10977] but found [8713]
at TestCases.E2E_negativeTest.placeOrder(E2E_negativeTest.java:107)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- placeOrder -->
</testsuite> <!-- TestonEdge -->
